### ISSUE TITLE
Add Effect: Save Game

### ISSUE THREAD

**Issue by DBWhale** [gta-chaos-mod/ChaosModV#792]
"Save Game"
(it can also be called "Quicksave" or the like if you prefer) 

It would be one of those effects that would last until you quit the game. (if not that than a long time)

This effect, when activated, will "save" your game. Whenever you die, you will spawn back at the spot where the effect was activated instead of the nearest hospital.

**Comment by anto4440** [comment]
Thats actually called respawn anchor. Its just important that its kept in its seperate file, and doesnt get mixed with anything else. But I have to say, this would be too destructive if you get this as your first effect and lose several missions.

**Comment by Da-Best** [comment]
this could lead to a fail loop if you somehow die in a mission and you get teleported to where this effect started. With the new fail safe i dont think this effect will be in it seeing it isn't active anymore so you will just be forced to reset the game. If it is an effect it will probably be better if it was a long timed effect instead of a forever lasting event

**Comment by anto4440** [comment]
> this could lead to a fail loop if you somehow die in a mission and you get teleported to where this effect started. With the new fail safe i dont think this effect will be in it seeing it isn't active anymore so you will just be forced to reset the game. If it is an effect it will probably be better if it was a long timed effect instead of a forever lasting event

No, because you only respawn once, in fact the worst case scenario is that you lose a chunk of progress.
